api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -211,6 +211,13 @@ message JwtProvider {
   // Specify the clock skew in seconds when verifying JWT time constraint,
   // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
   uint32 clock_skew_seconds = 10;
+
+  // Enables JWT cache, its size is specified by *jwt_cache_size*.
+  // Only valid JWT tokens are cached.
+  bool enable_jwt_cache = 11;
+
+  // The unit is number of JWT tokens, default to 100.
+  uint32 jwt_cache_size = 12;
```

Comment:
We should group this config in a separate message `JwtCacheConfig`, and that can potentially have the size limit configurable in the future.

Reply:
Added `JwtCachedConfig` and move `jwt_cache_size` in it.